### PR TITLE
Add "Search RubyGems" extension

### PR DESCRIPTION
**Pull request by aleks**
## Description

Hello 👋 

I became a huge fan of Raycast recently but was a bit annoyed that i couldn't use it to look up information from [rubygems.org](https://rubygems.org/) - here is the fix for that! 🥳 

- Use **Search RubyGems** to quickly find gems you're looking for
- Use the **Copy gem string** command to copy Gemfile compatible strings (i.e. `gem 'rails', '6.1.4.1'`)
- Gain quick access to gem specific links like **documentation**, **bug trackers** and **source code**
- See gem **dependencies** directly in Raycast

Ruby-familiar developers should quickly understand what's going on, but here are a few screenshots to make it clear:

light-dark
gem-options
gem-info

## Type of change
- New extension

## Screencast
https://user-images.githubusercontent.com/86240/145280928-c10f4805-3510-4587-95ec-b9d36c7ee577.mov

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/search-rubygems/src/components/GemOptions.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <List navigationTitle={`Search RubyGems > ${gem.name}`}>
+      <List.Section title="Actions">
+        <List.Item
+          key="gem-name"
+          icon={Icon.ArrowRight}
+          title="Show details"
+          accessoryTitle="Dependencies and more"
+          actions={
+            <ActionPanel>
+              <PushAction
```
Comment (pernielsentikaer) [inline extensions/search-rubygems/src/components/GemOptions.tsx:19]: No icon added, please add an icon for PushAction

File: extensions/search-rubygems/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        results.map(gem => {
+          return <List.Item
+            key={gem.sha}
+            icon="list-icon.png"
+            title={gem.name}
+            accessoryTitle={`\u21E9 ${gem.downloads.toLocaleString()} | ${gem.version}`}
+            subtitle={gem.info}
+            actions={
+              <ActionPanel>
+                <PushAction
```
Comment (pernielsentikaer) [inline extensions/search-rubygems/src/index.tsx:37]: No icon added, please add an icon for PushAction

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @aleks ⭐ 

Thanks for your contribution, I left a few minor corrections that should be addressed 💪 

Looking forward to test it out again 💯 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/search-rubygems/src/components/GemOptions.tsx (hunk 1 above)
Comment (aleks) [reply extensions/search-rubygems/src/components/GemOptions.tsx:19]: @pernielsentikaer fixed!

File: extensions/search-rubygems/src/index.tsx (hunk 2 above)
Comment (aleks) [reply extensions/search-rubygems/src/index.tsx:37]: @pernielsentikaer fixed!

**Comment by aleks** [comment]
@pernielsentikaer thanks for the review! 🥳 I added icons on both `PushAction`s and updated the titles a bit. Let me know if you need anything else! 🤘

File: extensions/search-rubygems/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "search-rubygems",
+  "title": "Search RubyGems",
```
Comment (pernielsentikaer) [inline extensions/search-rubygems/package.json:4]:
```suggestion
  "title": "Search Documentation",
  "subtitle": "RubyGems",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @aleks 👋 

Just found a small one, could you correct this, when it's done I think we're ready to merge 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi @aleks 🔥 

Thanks for the update! I'm going to accept it and then will my colleague do a final review before it's finally merged 🚀

**Review by mattisssa — APPROVED** [review]

**Pull request merged**